flytekit/plugins/__init__.py

```diff
@@ -46,3 +48,9 @@
     ],
     [hmsclient]
 )
+
+_lazy_loader.LazyLoadPlugin(
+    "pytorch",
+    ["torch>=1.0.0,<1.6.0"],
```

Comment:
Is there a breaking change after 1.6? generally speaking, if they are following semantic versioning, up to 2.0.0 should be fine, right?